modules/ROOT/pages/changelog.adoc

```diff
@@ -5,6 +5,22 @@
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
 
+== 6.7.2 - 2023-10-25
+
+=== Fixed
+* Applying heading formatting to the content of the `summary` element extended its application to the content of the parent `details` element.
+//TINY-10154
+* The function `getModifierState` did not work on events passed through the editor as expected.
+//TINY-10263
+* List items containing a nested list between two text caused some list operations to fail.
+//TINY-10268
+* css loading in `toolbardrawertoggletest` flickering.
```

Comment:
`toolbardrawertoggletest` what is that? It should likely state that the theme loading is no longer async as the changelog item in the tinymce repo? So maybe `Reduced UI flickering when reloading the editor by making the theme initialization sync instead of async`

Reply:
```suggestion
* Removed use of async for editor rendering which caused visual blinking when reloading the editor in-place.
```

Reply:
@spocke update the below, thanks for the catch